.github/ISSUE_TEMPLATE.MD

```diff
@@ -0,0 +1,39 @@
+<!--
+If you are reporting a new issue, make sure that we do not have any duplicates
+already open. You can ensure this by searching the issue list for this
+repository.
+For more information about reporting issues, see
+https://github.com/budhrg/vagrant-service-manager/blob/master/CONTRIBUTING.adoc#submitting-issues.
+-->
+
+**OS details:**
+
+
+**Provider:**
+VirtualBox and/or Libvirt
+
```

Comment:
@LalatenduMohanty This one is for ADB issue template and until we support HyperV for ADB, it don't think its good idea.